src/snyk/common/error/errorReporter.ts

```diff
@@ -0,0 +1,108 @@
+import * as Sentry from '@sentry/node';
+import { Contexts, Transport, TransportClass } from '@sentry/types';
+import { Configuration, IConfiguration } from '../configuration/configuration';
+import { SnykConfiguration } from '../configuration/snykConfiguration';
+import { ILog } from '../logger/interfaces';
+import { Platform } from '../platform';
+import { IVSCodeEnv } from '../vscode/env';
+import { OnUncaughtException } from './integrations/onUncaughtException';
+
+type SentryEnvironment = 'development' | 'preview' | 'production';
+
+export class ErrorReporter {
+  private static readonly eventQueueTimeoutMs = 1000;
+
+  /**
+   * 'OnUncaughException' integration must be ignored since it causes Sentry to permanently fail during runtime. Default Sentry behaviour is to call process.exit() for uncaught errors. This is being prevented by VS Code and is logged as:
```

Comment:
Please break the line :)

Reply:
made a break after "errors" as agreed in DM :) ✔️